bitcoin/src/psbt/mod.rs

```diff
@@ -407,7 +407,8 @@ impl Psbt {
                 let sighash = cache.legacy_signature_hash(input_index, spk, hash_ty.to_u32())?;
                 // TODO: After upgrade of secp change this to Message::from_digest(sighash.to_byte_array()).
```

Comment:
I do not get the meaning of this TODO, maybe is outdated?

Reply:
Can we address it now that we have a rust-secp release?

Reply:
Oh nice, crazy that I missed this, it literally shows up in the diff of the upgrade. Added as a new additional patch.